Promoted Helm chart version to 1.0.0 (#2)
WebSpellChecker/wproofreader-helm: README.md

## Basic installation

The chart uses `nodeAffinity` for mounting Persistent Volume of type `local`.
This also allows the user to specify which node will host the WProofreader Server 
on a cluster (even a single-node one).

To assign this role to a node, one has to attach a label to it. It can be whatever you want it to be,
e.g. `proofreader.your-company.com/app`:
```shell
kubectl label node <name-of-the-node> proofreader.company-domain.com/app=
```
Note that `=` is required, but the value after it is not important (empty in this example).

Keep in mind that your custom label has to be either updated in `values.yaml`
(`affinityLabel` key, recommended), or passed to `helm` calls using 
`--set affinityLabel=proofreader.company-domain.com/app`.

Now, the chart can be installed the usual way using all the defaults:
The Chart can be installed the usual way using all the defaults:
```shell
git clone https://github.com/WebSpellChecker/wproofreader-helm.git
cd wproofreader-helm

> helm install --create-namespace --namespace wsc wsc-app-5-x-x wproofreader --set affinityLabel=proofreader.company-domain.com/app 

Reviewer: do you think there is no sense to keep this naming/versioning as a default?

Author: In my opinion, it is better to have it the way it is. 
The Helm release name is the part of naming convention here and the resources that will be created will contain it in their names. 
So having a version there will confuse more when using `helm update` command or `--set image.tag`, for example. 
But depending on deployment strategy it is also possible to have a version there.

**Recommended approach:** Using an existing PVC is the recommended way because it ensures that your data will persist even if the Chart is uninstalled. This approach offers a reliable method to maintain data integrity and availability across deployments.

However, please note that provisioning the Persistent Volume (PV) and PVC for storage backends like NFS is outside the scope of this Chart. You will need to provision the PV and PVC separately according to your storage backend's documentation before using the `dictionaries.existingClaim` parameter.

## Use in production

For production deployments, it is highly recommended **to specify resource requests and limits for your Kubernetes pods**. This helps ensure that your applications have the necessary resources to run efficiently while preventing them from consuming excessive resources on the cluster which can impact other applications.
This can be configured in the `values.yaml` file under the `resources` section.

### Recommended resource requests and limits

> Below are the recommended resource requests and limits for deploying WProofreader Server v5.34.x with English (en_US, en_GB, en_CA, and en_AU) and including features such as a style guide, spelling autocorrect, Named-Entity Recognition (NER), and text autocomplete. These values reflect the minimum requirements for running WProofreader Server in a production environment.

Reviewer: ```suggestion
Below are the recommended resource requests and limits for deploying WProofreader Server v5.34.x with enabled English dialects (en_US, en_GB, en_CA, and en_AU) for spelling & grammar check using the English AI language model for enhanced and more accurate proofreading. It also includes such features as a style guide, spelling autocorrect, named-entity recognition (NER), and text autocomplete suggestions (text prediction). These values represent the minimum requirements for running WProofreader Server in a production environment.
```

Author: Applied

> Depending on your specific needs and usage patterns, especially if deploying AI language models for enhanced proofreading, you may need to adjust these values to ensure optimal performance and resource utilization.

Reviewer: ```suggestion
**Note:** Depending on your specific needs and usage patterns, especially when deploying AI language models for enhanced proofreading in other languages, you may need to adjust these values to ensure optimal performance and resource utilization. Alternatively, you can choose the bare-minimum configuration without AI language models. In this case, only algorithmic engines will be used to provide basic spelling and grammar checks.
```

Author: Applied

However, please note that provisioning the Persistent Volume (PV) and PVC for storage backends like NFS is outside the scope of this Chart. You will need to provision the PV and PVC separately according to your storage backend's documentation before using the `dictionaries.existingClaim` parameter.

## Use in Production

For production deployments, it is highly recommended to specify resource requests and limits for your Kubernetes pods. This helps ensure that your applications have the necessary resources to run efficiently while preventing them from consuming excessive resources on the cluster, which can impact other applications.
This can be configured in the `values.yaml` file under the `resources` section.

### Recommended Resource Requests and Limits

Below are the recommended values for resource requests and limits. These values are based on the minimum requirements for running the WProofreader Server in a production environment. You may need to adjust these values based on your specific requirements and usage patterns, especially when deploying AI-models, to ensure optimal performance and resource utilization.

> ```yaml

Author: Updated.

> There are several options for how to gather needed details:

Reviewer: ```suggestion
There are several ways to gather necessary details:
```

1. Get the values (user-configurable options) used by Help to generate Kubernetes manifests:
```shell
helm get values --all --namespace wsc wproofreader-app > wproofreader-app-values.yaml
```
where `wsc` is the namespace and `wproofreader-app` – the name of your release, 
and `wproofreader-app-values.yaml` – name of the file the data will be written to.

2. Extract the full Kubernetes manifest(s) as follows:
```shell
helm get manifest --namespace wsc wproofreader-app > manifests.yaml
```

> If, for any reason, you do not have access to `helm`, same can be accomplished using 

Reviewer: ```suggestion
If you do not have access to `helm`, same can be accomplished using 
```

> `kubectl`. To get manifests for all resources in `wsc` namespace, use:

Reviewer: ```suggestion
`kubectl`. To get manifests for all resources in the `wsc` namespace, run:
```

```shell
kubectl get all --namespace wsc -o yaml > manifests.yaml
```

> 3. Get the logs of all the wsproofreader-app pods in the `wsc` namespace:

Reviewer: ```suggestion
3. Retrieve the logs of all `wsproofreader-app` pods in the `wsc` namespace:
```
